### ISSUE TITLE
ng: support "scan" simulations

### ISSUE THREAD

**Issue by climbcat** [McStasMcXtrace/McWeb#28]
- backend: 
    - recursively rename any mcstas.sim to mccode.sim
    - recursively locate mccode.sim files
    - parse mccode.sim for data files, which may have any name
    - plot all ".dat" files (of any name) in scan subdirs of data-folder/mcstas
    - reflect datastructure in models.SimRun

- frontend: 
    - implement a new view which looks a lot like status.html, but is suited for viewing scan simulations
    - monitors can be viewed in parallel for each scan

**Comment by climbcat** [comment]
this is now supported by the backend

**Comment by climbcat** [comment]
the frontend part of this is also done, the task is done

**Comment by willend** [comment]
Most is done - but simfiles must be able to have any name - see separate issue
